frank-flow/src/frontend/src/app/shared/enums/file-type.enum.ts

```diff
@@ -1,4 +1,5 @@
 export enum FileType {
   JSON,
```

Comment:
Maybe we should rename the enums in FileType, because it was originally meant to define the difference between a xml dom structure and a json structure. Or at least delete the JSON type.

Reply:
I think this can get fixed in the while refactor after the MVP.